ZenPacks/zenoss/Docker/modeler/plugins/DockerCollector.py

```diff
@@ -45,7 +45,7 @@ def process(self, device, results, log):
         oms = []
         for line in results.splitlines()[2:]:
             bits = [x.strip() for x in \
-                filter(lambda x: x.strip(), line.split("  "))]
```

Comment:
Could we simplify it like this : `[x.strip() for x in line.split(' ' * 3) if x.strip()]` ?